docs/book/usage/setup.md

```diff
@@ -78,13 +23,22 @@ Windows. The Ubuntu version most likely works on other Linux distributions as we
 | MacOS   | Intel 64bit           | [`macos-x86_64`](https://storage.googleapis.com/mysten-walrus-binaries/walrus-mainnet-latest-macos-x86_64)                   |
 | Windows | Intel 64bit           | [`windows-x86_64.exe`](https://storage.googleapis.com/mysten-walrus-binaries/walrus-mainnet-latest-windows-x86_64.exe)       |
 
-### Install via script {#nix-install}
+```admonish tip
+Our latest Walrus binaries are also available on Walrus itself, namely on
+<https://bin.wal.app>, for example,
+<https://bin.wal.app/walrus-mainnet-latest-ubuntu-x86_64>.
+Because of DoS protection, it might not be possible to download the binaries with
+`curl` or `wget`.
+```
 
-To download and install `walrus` to your `"$HOME"/.local/bin` directory, run one of the following
-commands in your terminal then follow on-screen instructions. If you are on Windows, see the
-[Windows-specific instructions](#windows-install) or the [`suiup` installation](#suiup-install) (experimental)
+## Install via script {#nix-install}
```

Comment:
Question: What is the rationale for making all installation options H2 headings (instead of the previous H3)?